book.bib

```diff
@@ -520,18 +520,24 @@ @article{Chao1984
     year = {1984}
 }
 
-@article{LahtiL2014,
+@article{Lahti2014,
     author = {Lahti L et al.},
     title = {Tipping elements in the human intestinal ecosystem},
     journal = {Nature Communications},
+    volume = {2014},
+    issue = {5},
+    pages = {1-10},
     doi = {https://doi.org/10.1038/ncomms5344},
     year = {2014}
 }
 
-@article{SaloS2021,
+@article{Salosensaari2021,
     author = {Salosensaari et al.},
```

Comment:
sorry.. include the full author list..

Reply:
and if you in some cases need to write et al. due to huuge number of authors, it would be "and others" in bibtex markup